.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+name: OpenWISP Utils CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install python dependencies
+      run: |
+        pip install -U pip wheel setuptools
+        pip install -U -r requirements-test.txt
+
+    - name: Install openwisp-utils
+      run: |
+        pip install -e .[qa,rest]
+        pip install ${{ matrix.django-version }}
+
+    - name: QA checks
+      run: |
+        ./run-qa-checks
```

Comment:
Can't this be `run: ./run-qa-checks` ? :smile:

Reply:
right, one line is better in this case